src/components/AppGrid.vue

```diff
@@ -236,7 +236,7 @@ main {
   position: absolute;
   display: grid;
   grid-auto-flow: row dense;
-  @include colors(20, 100);
+  @include colors(50, 100);
```

Comment:
I don't think the change can be this simple. The color stops are now too similar and you can't see the difference between them well.